src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/HarmonicSchwarzschild.hpp

```diff
@@ -0,0 +1,935 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <pup.h>
+
+#include "DataStructures/CachedTempBuffer.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/AnalyticSolution.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Solutions.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/Literals.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
```

Comment:
(nitpick) you're missing some includes, e.g. `TensorExpressions` in the cpp. Nothing to hold up this PR, but please do another pass over the included when you get the chance. You can delete all the IWYU comments if you want (we don't actively use IWYU at the moment).

Reply:
I'll do another pass on the includes!

Regarding `TensorExpressions` specifically, though, mine and @nilsdeppe 's thought was to have `TensorExpressions` be considered part of the `Tensor` interface in that when you include `Tensor.hpp`, with it comes all of `TensorExpressions`. We currently just include all of the `TensorExpression` header files in `Tensor.hpp` to do this. This is one way of not having to have people include each individual TE file that they need.

Reply:
Sounds good, I believe Tensor.hpp is missing though